### DISCUSSION TITLE
Safari cannot open the page. Error: "cannot parse response

### DISCUSSION THREAD

**Discussion by DeaiLy** [YOURLS/YOURLS#3757] [Q&A]
It redirects from android and computer without any problem, but when clicked from ios device "Safari cannot open the page. Error: "cannot parse response" error, what is the reason, can you help me solve it?

**Comment by dgw** [comment]
Seems like there should be more to the error than just "cannot parse response". Reports of similar issues with other sites in the past year-ish include an error code of some kind, e.g. `The error is: "cannot parse response" (NSURLErrorDomain:-1017)`

**Reply by DeaiLy** [reply]
unfortunately it does not contain an error code. it just says that and does not redirect

**Reply by dgw** [reply]
Hmm. I don't own any Apple stuff, so I can't test. :/
